### PR TITLE
Paginate trees history

### PR DESCRIPTION
**Pull request by SantiSite**

### CODE REVIEW

File: src/components/providers/UserProvider.tsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ interface Props {
 const UserProvider: FC<Props> = ({ children }: Props) => {
   const firebase = useFirebase();
   const [acceptsEmail, setAcceptsEmail] = useState<boolean | undefined>(
-    false,
```
Comment (odarbelaeze) [inline src/components/providers/UserProvider.tsx:16]: Is this change intentional?

Comment (SantiSite) [reply src/components/providers/UserProvider.tsx:16]: Yes, since it generated an error in the pagination, but I will set it back to false, and perform the infinity scroll.

**Comment by dani-vh** [comment]
@odarbelaeze, please take a look and let me know if there's something else you want to tweak. I left the minimal implementation that was defined since the begining, should be enough to accomplish with the MVP.

**Comment by odarbelaeze** [comment]
@dani-vh looks good, the only thing I would change is to take off the `scrollableDiv`, assuming the infinite scroll would work just with the scrolling of the page otherwise.

**Review by odarbelaeze — APPROVED** [review]
:ship::it:

**Review by odarbelaeze — APPROVED** [review]
